package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Make sure to delete this extra package-lock.json

Reply:
forgot to commit the deletion